cicd/pipelines/using-pipelines-as-code.adoc

```diff
@@ -0,0 +1,58 @@
+:_content-type: ASSEMBLY
+[id="using-pipelines-as-code"]
+= Using Pipelines as Code
+include::_attributes/common-attributes.adoc[]
+:context: using-pipelines-as-code
+
+toc::[]
+
+:FeatureName: Pipelines as Code
+include::snippets/technology-preview.adoc[]
+
+[role="_abstract"]
+With {pac}, cluster administrators and users with the required privileges can define pipeline templates as part of source code Git repositories. When triggered by source code push or a pull request for the configured Git repository, the feature runs the pipeline and reports status.   
+
+[id="pac-key-features"]
+== Key features
+{pac} supports the following features:
```

Comment:
I would recommend rephrasing the following list to focus on the user's actions and goals. For example: https://github.com/openshift/openshift-docs/pull/47816#discussion_r924569869

Reply:
@rolfedh : While I agree that we write user stories and should emphasize user's actions, sometimes outlining the key features of a product can be beneficial while introducing the product or a component. For example, see [Pipelines - Key features](https://docs.openshift.com/container-platform/4.10/cicd/pipelines/understanding-openshift-pipelines.html#op-key-features) and [GitOps - Key features](https://docs.openshift.com/container-platform/4.10/cicd/gitops/understanding-openshift-gitops.html#key-features_understanding-openshift-gitops). In both occasions, the content was not written by me, and reviewed and approved.
If we are absolutely against outlining key features while introducing something, then I think we should create a separate JIRA task for that and weed out all such instances.
For this iteration of PaC docs, I would request you to agree on the key features, mostly because those features show how cleverly PaC automates many of the chores. Because of the emphasis of those features being automated capabilities, it is difficult to frame them in the form of a user-based action.